app/src/hooks/activity-value-form/use-activity-form.ts

```diff
@@ -28,41 +60,39 @@ const useActivityForm = ({
           ...targetActivityValue.activityData,
           dataQuality: targetActivityValue?.dataSource?.dataQuality,
           sourceReference: targetActivityValue?.dataSource?.notes,
-          CH4EmissionFactor: targetActivityValue?.activityData?.ch4_amount,
-          CO2EmissionFactor: targetActivityValue?.activityData?.co2_amount,
-          N2OEmissionFactor: targetActivityValue?.activityData?.n2o_amount,
-          activityDataAmount: 0,
-          activityDataUnit: null,
-          emissionFactorType: "",
-          totalFuelConsumption: "",
-          totalFuelConsumptionUnits: "",
-          co2EmissionFactorUnit: "",
-          n2oEmissionFactorUnit: "",
-          ch4EmissionFactorUnit: "",
+          CH4EmissionFactor:
+            methodologyName === "direct-measure"
+              ? targetActivityValue?.activityData?.ch4_amount
+              : extractGasAmount("CH4", targetActivityValue).amount,
+          CO2EmissionFactor:
+            methodologyName === "direct-measure"
+              ? targetActivityValue?.activityData?.co2_amount
+              : extractGasAmount("CO2", targetActivityValue).amount,
+          N2OEmissionFactor:
+            methodologyName === "direct-measure"
+              ? targetActivityValue?.activityData?.n2o_amount
+              : extractGasAmount("N2O", targetActivityValue).amount,
+          emissionFactorType: targetActivityValue.metadata?.emissionFactorType, // TODO confirm the source of this value
+          totalFuelConsumption:
+            targetActivityValue?.metadata?.totalFuelConsumption, // TODO confirm the source of this value
```

Comment:
This is an input provided by the user (input needs to be generated from the manual input hierarchy data for the chosen methodology).
I would put this into `activityData.totalFuelConsumption` instead of in `metadata` as it is used for calculations. Generally everything used in the formulas should be in `activityData`, everything that's exclusively used to restore the state of forms or other UI elements etc. should be in `metadata`.

Reply:
This means having the emissionsFactorType in metadata as shown above is correct :+1: